simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -367,6 +370,43 @@ public void live_reloading() {
                 .run();
     }
 
+    @Test
+    public void uncommon_flakes() {
+        servers = servers(
+                SimulationServer.builder()
+                        .serverName("fast0")
+                        .simulation(simulation)
+                        .handler(h -> h.response(respond500Every(100)).responseTime(Duration.ofNanos(1000)))
+                        .build(),
+                SimulationServer.builder()
+                        .serverName("fast1")
+                        .simulation(simulation)
+                        .handler(h -> h.response(respond500Every(100)).responseTime(Duration.ofNanos(1000)))
+                        .build());
+
+        result = Benchmark.builder()
+                .requestsPerSecond(1000)
+                .sendUntil(Duration.ofSeconds(10))
+                .clients(10, i -> strategy.getChannel(simulation, servers))
+                .simulation(simulation)
+                .abortAfter(Duration.ofSeconds(10))
+                .run();
+    }
+
+    private Function<SimulationServer, Response> respond500Every(int reqs) {
```

Comment:
I think it's fine to drop this as a default method into the HandlerBuilder0 interface or whatever it was

Reply:
500 is an odd case, I'd like to test what happens when there's a connect or socket exception, but that doesn't appear to be implemented yet. I don't think most of our limiters actually care about 500.

Reply:
Feel free to just return a response containing that exception! No need to stick to status codes

Reply:
It doesn't look like most of our limiters care about exceptions yet, I'll leave this as is for the time being